contributing/development/core_and_modules/internal_rendering_architecture.rst

```diff
@@ -542,13 +541,12 @@ Global illumination
 
 .. note::
 
-    VoxelGI and SDFGI are only available in the Forward+ backend, not the
-    Forward Mobile or Compatibility methods.
+    VoxelGI and SDFGI are only available in the Forward+ renderer, not the
+    Forward Mobile or Compatibility renderers.
```

Comment:
I intentionally left some references to "Forward Mobile" on the Internal Rendering Architecture page alone, because I thought "Forward Mobile" was still a valid name for this renderer, just not the one we should use in most places. For example, the heading on this page is still called "Forward Mobile".

I'm not against fully standardizing, though. If all the references on this page change, we should probably change the descriptions of the renderers in `Renderers` and `List of Features` to not say that Mobile is also called "Forward Mobile".

Reply:
I went ahead with the complete standardization in the manual.